tests/integration/templatetags/test_oscar_thumbnail.py

```diff
@@ -59,6 +60,26 @@ def test_oscar_thumbnail_tag_as_context_value(self):
     def test_oscar_thumbnail_sizes(self):
         self._test_oscar_thumbnail_tag_sizes()
 
+    @patch('oscar.templatetags.image_tags.ThumbnailNode._render')
+    def test_ThumbnailNode_doesnt_raise_if_OSCAR_THUMBNAIL_DEBUG_is_false(self, render_mock):
+        render_mock.side_effect = ValueError()
+        with override_settings(OSCAR_THUMBNAIL_DEBUG=True):
+            with self.assertRaises(ValueError):
```

Comment:
This check (for an exception) contradicts what the test name says it's doing - think there should be two tests, one for each setting state.